src/routes/+layout.svelte

```diff
@@ -61,65 +55,67 @@
 				<span class="primary-action__icon">✨</span>
 			</a>
 		</hgroup>
-		<HR />
 	</header>
 
-	<aside
-		id="aside"
-		class="layout__aside {isExpanded ? 'layout__aside--expanded' : ''}"
-		role="region"
-		aria-labelledby="hamburger"
-	>
-		<ul class="aside__ul">
-			{#if data.user}
-				<li class="aside__li">
-					<a class="aside__a" href="/user/{data.user.id}/" aria-disabled="true">
-						<strong class="aside__strong">
-							{data.user.nickname}
-						</strong>
-					</a>
-				</li>
-				<li class="aside__li">
-					<a class="aside__a" href="/user/{data.user.id}/drafts/" aria-disabled="true">Drafts</a>
-				</li>
+	{#if isExpanded}
+		<aside
+			id="aside"
+			class="layout__aside"
+			role="region"
+			aria-labelledby="hamburger"
+			transition:slide="{{ duration: 50 }}"
```

Comment:
This component animation is a bit glitchy so I made the transition faster so it's less noticeable.